_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -171,7 +179,16 @@ public function page_render() {
 							</a>
 						</p>
 						<p>
-							<?php esc_html_e( 'We’re a distributed company with over 875 Automatticians in more than 67 countries speaking at least 83 different languages. Our common goal is to democratize publishing so that anyone with a story can tell it, regardless of income, gender, politics, language, or where they live in the world.', 'jetpack' ); ?>
+							<?php
+							echo esc_html(
+								sprintf(
+									/* translators: first placeholder is the number of Automattic employees. The second is the number of countries of origin. */
+									__( 'We’re a distributed company with over %1$s Automatticians in more than %2$s countries speaking at least 83 different languages. Our common goal is to democratize publishing so that anyone with a story can tell it, regardless of income, gender, politics, language, or where they live in the world.', 'jetpack' ),
```

Comment:
Should you fetch the number of languages spoken as well?

Reply:
Good catch